src/controller/age.controller.js

```diff
@@ -17,7 +18,9 @@ router.get("/howold", async (req, res) => {
   const today = new Date();
   const birthDate = new Date(dob);
   if (birthDate == "Invalid Date") {
-    return res
+    return res.status(400).json({
+      error: "Only date format yyyy-mm-dd is allowed"
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/armerillo/ql-task/pullRequest?prid=9974416)
```suggestion
      error: "Only date format yyyy-mm-dd is allowed",
```